test/unit/math/mix/fun/sd_test.cpp

```diff
@@ -6,13 +6,21 @@ void expect_sd(const T& m, const stan::test::ad_tolerances& tols
   auto f = [](const auto& x) { return stan::math::sd(x); };
   Eigen::VectorXd v(m.size());
   Eigen::RowVectorXd rv(m.size());
+  std::vector<double> sv(m.size());
   for (int i = 0; i < m.size(); ++i) {
     v(i) = m(i);
     rv(i) = m(i);
+    sv[i] = m(i);
   }
+
   stan::test::expect_ad(tols, f, v);
   stan::test::expect_ad(tols, f, rv);
   stan::test::expect_ad(tols, f, m);
+  stan::test::expect_ad(tols, f, sv);
+
```

Comment:
We're missing autodiff tests for `std::vector<mat<var>>`.

I checked the docs and it's not doc'd (so not exposed to users yet). `log_sum_exp` and `log_softmax` have tests though.

I guess I'll just copy those eventually but I tried a couple simple ones and they were failing for me today so that's something for the future.

Reply:
I updated the `sd` implementation to be vectorized in the way that `apply_vector_unary` supports (so `std::vector`, `vector`, `row_vector`, and `matrix` all get passed in as an Eigen vector map, and `std::vector` s of any of those work as well) and added the test framework changes.

Reply:
Nice!